1907-05-09.xml

```diff
@@ -0,0 +1,6010 @@
+                    schools of this nature."</p>
+                <p>THE ANGLOPHÕBE PRESS. In conclusion Lord Cromer says it would be manifestly
+                    unjust to criticise too severely the proceedings of the Assembly Some of the
+                    points to which attention was directed are well worthy of consideration, and
+                    much consideration, he states, they will receive. On the other hand, apart from
+                    the question of the extent to which the Assembly is really repre. sensitive of
+                    the views of the mass of the Egyptians, it is more than doubtful whether he
+                    proposals put forward actually represent the views of the majority of the
+                    Assembly itself. "It is a notorious fact, adds Lord Cromer, that there is a very
+                    large body, probably constituting a majority, who are animated by very friendly
+                    feelings towards the present régime, and who are sufficiently well acquainted
+                    with the conditions of their country to know that any progress in the direction
+                    of Parliamentary institutions, ete, must be gradual These, however, were to a
+                    great extent terrorised by the Anglophobe Press, and, more. over, pressure of
+                    various sorts, was I am given to understand, brought to bear upon them."</p>
+                <p>ROYAL DUBLIN FUSILIERS.</p>
+                <p>Attention has been drawn in a London contemporary to the alleged bloek in promo:
+                    tion in the Royal Dublin Fusiliers---the 1st battalion of which is in Egypt and
+                    the 2nd battalion at Fermoy-owing to the retention of serjeants on the militia
+                    staf beyond the ordinary period. It is pointed out that there are no fewer than
+                    a score of non-commissioned officers, of from 25 to 29 years' service, doing
+                    duty with the staffs of the militia battalions of the Dublins, the natural
+                    result being that promotion is at a standstill amongst the senior non coins. For
+                    instance one non-com. has been lance sergeant for eight years. If these
+                    statements be correct, it is pretty near time that something was done to remedy
+                    this state of affairs.</p>
+                <p>
+                    <hi rend="bold">BERLITZ SCHOOLS OF LANGUAGES</hi>
+                </p>
+                <p>
+                    <hi rend="bold">326 BRANCHES. </hi>
+                </p>
+                <p>
+                    <hi rend="bold">French, German, Italian, Greek, Arabic, etc. Private Lessons
+                        Residence Lessons, taught by </hi>
+                    <hi rend="bold">Native Masters</hi>
+                </p>
+            </div>
+            <cb n="5"/>
+            <div type="section">
```

Comment:
this div needs to be divided into items, corresponding to the separate items on the page